src/main.rs

```diff
@@ -117,10 +126,29 @@ pub enum SubCmd {
         /// The name of the service to remove.
         #[clap(long)]
         service_name: Option<String>,
-        /// Set this flag to keep the nodes data and log directories.
+        /// Set this flag to keep the node's data and log directories.
         #[clap(long)]
         keep_directories: bool,
     },
+    /// Run a local network.
+    ///
+    /// This will run safenode processes on the current machine to form a local network. A faucet
+    /// service will also run for dispensing tokens.
+    ///
+    /// Paths can be supplied for safenode and faucet binaries, but otherwise, the latest versions
+    /// will be downloaded.
+    #[clap(name = "run")]
+    Run {
+        /// The number of nodes to run.
+        #[clap(long)]
+        count: Option<u16>,
```

Comment:
Either set a default with `clap` or make sure it's documented what happens when `--count` isn't used.
```suggestion
        #[clap(long, default_value_t = DEFAULT_NODE_COUNT)]
        count: u16,
```